### PR TITLE
updates from experience and feedback

### PR DESCRIPTION
**Pull request by spucci**

### CODE REVIEW

File: Processes/Cycle Plannning.md [hunk 1]
```diff
@@ -26,14 +27,14 @@ Checklist for whoever is running cycle planning to go through during the meeting
 - highlight current company goals and product strategy
 - report on key metric status (insights delivered)
 - close sprint
-	- all non-done tickets will automatically move to new sprint
+	- move all non-done tickets will to new sprint (with discussion)
```
Comment (dodeca) [inline Processes/Cycle Plannning.md]: typo w/ an extra 'will'?

**Comment by mpetrovich** [comment]
cc @nextbigsoundinc/all

**Comment by mpetrovich** [comment]
👍

**Comment by samirrayani** [comment]
👍

**Pull request merged**